src/components/PlaybackControls/index.tsx

```diff
@@ -159,13 +192,20 @@ const PlayBackControls = ({
                 disabled={loading || isEmpty}
             />
             <div className={styles.time}>
-                <p>
-                    {displayTimes.roundedTime}{" "}
-                    <span className={styles.lastFrameTime}>
-                        / {displayTimes.roundedLastFrameTime}{" "}
-                        {displayTimes.unitLabel}
-                    </span>
-                </p>
+                <InputNumber
+                    // key is necessary to re-render this component and override user input
+                    key={displayTimes.roundedTime}
+                    size="small"
+                    value={displayTimes.roundedTime}
+                    onChange={handleTimeInputChange}
+                    onPressEnter={handleTimeInputPressEnter}
```

Comment:
not sure if this is required, but some users may expect "tab" to also enter the value?  or any defocus event?  or does defocus cancel the text entry?

Reply:
Right, good points, I just enabled Tab to commit time, but defocus/ESC is going to require a bit more work (since I have to figure out the best way to clear the text entry and make the component re-render... let me know if you know an obvious way). Ok if we do that on a separate ticket?

Reply:
Another quirk I just found that I'd need to figure out with defocus is, it seems like if I make an `onBlur` handler it doesn't detect clicking away if I click on somewhere on the viewer canvas (clicking on elsewhere in the playback controls, or the left or right side panels, triggers it but not the viewer canvas 🙄 )

Reply:
Actually I just found out that the defocus to clear entry is already automatically implemented without an explicit `onBlur` handler, except it faces the same issue above which is clicking away on the viewer canvas doesn't do anything.

Reply:
Okay, enabled Esc to cancel input!